pages/exercises/[lessonSlug].tsx

```diff
@@ -9,36 +9,21 @@ import Error, { StatusCode } from '../../components/Error'
 import LoadingSpinner from '../../components/LoadingSpinner'
 import AlertsDisplay from '../../components/AlertsDisplay'
 import NavCard from '../../components/NavCard'
-import ExercisePreviewCard, {
-  ExercisePreviewCardProps
-} from '../../components/ExercisePreviewCard'
+import ExercisePreviewCard from '../../components/ExercisePreviewCard'
 import { NewButton } from '../../components/theme/Button'
 import ExerciseCard, { Message } from '../../components/ExerciseCard'
 import { ArrowLeftIcon } from '@primer/octicons-react'
 import GET_EXERCISES from '../../graphql/queries/getExercises'
 import styles from '../../scss/exercises.module.scss'
 
-const exampleProblem = `const a = 5
-a = a + 10
-// what is a?`
-
-const mockExercisePreviews: ExercisePreviewCardProps[] = [
-  { moduleName: 'Variables', state: 'ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
-  { moduleName: 'Variables', state: 'ANSWERED', problem: exampleProblem }
-]
-
 const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
   queryData
 }) => {
   const { lessons, alerts, exercises } = queryData
   const router = useRouter()
   const [exerciseIndex, setExerciseIndex] = useState(-1)
+  const [userAnswers, setUserAnswers] = useState<Record<number, string>>({})
```

Comment:
We store the user's answers on the frontend for now. They go away when you refresh the page. This is good for testing. In the next pull request, we'll get the user's answers from the backend.